pkg/controller/attach_detach.go

```diff
@@ -63,12 +63,12 @@ Attach Detach controller is responsible for
 	3. detaching volume upon deletions marked with certain annotations
 */
 type ReconcileAttachDetach struct {
-	logger                logr.Logger
-	crdDetacher           CrdDetacher
-	cloudDiskAttacher     CloudDiskAttachDetacher
-	controllerSharedState *SharedState
-	stateLock             *sync.Map
-	retryInfo             *retryInfo
+	logger            logr.Logger
+	crdDetacher       CrdDetacher
+	cloudDiskAttacher CloudDiskAttachDetacher
+	stateLock         *sync.Map
+	retryInfo         *retryInfo
+	*sharedState
```

Comment:
Place anonymous fields at the beginning rather than the end.

Reply:
Done.